pkg/apis/compliance/v1alpha1/profile_types.go

```diff
@@ -4,6 +4,13 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
+// PlatformTypeLabel specifies what kind of platform (node,platform)
+// this Profile or a TailoredProfile targets
+const PlatformTypeLabel = "compliance.openshift.io/platform-type"
```

Comment:
nit: `platform-type: platform` sounds quite redundant; what about `scan-type`?

Reply:
Oh, naming :-) Since the label is about Profile and not scans, what about `profile-type` or `profile-kind` or even more generic `target-type` ?

Reply:
Given we would rename `PlatformLabel` below to `compliance.openshift.io/product`, what about `product-type` ?